cdap-docs/developers-manual/source/building-blocks/datasets/fileset.rst

```diff
@@ -130,3 +130,7 @@ See the Apache™ Twill®
 `API documentation <http://twill.incubator.apache.org/apidocs/org/apache/twill/filesystem/Location.html>`__
 for additional information about the ``Location`` abstraction.
 
+.. rubric::  Examples of Using FileSet Dataset
+
+- For an example that **demonstrates the use of time-partitioned File sets,** see the :ref:`Stream Conversion
```

Comment:
I think here we should point to the FileSetExample for an example of how to use FileSet; and then say: In version 2.7, we added a new experimental dataset, the TimePartitionedFileSet. For an example that demonstrates... (your text that is already here)